src/lib/object.c

```diff
@@ -335,6 +336,13 @@ CK_RV object_find_init(session_ctx *ctx, CK_ATTRIBUTE_PTR templ, CK_ULONG count)
     token *tok = session_ctx_get_token(ctx);
     assert(tok);
 
+    session_ctx_delete_tobject_list(ctx);
+
+    rc = init_tobjects(tok);
```

Comment:
We can probably drop the initial init call to make token initialization quicker:
in `src/lib/db.c` line 640 delete the following statement:
```C
        rc = init_tobjects(t);
        if (rc != SQLITE_OK) {
            goto error;
        }
```

Reply:
Hi William. that call is already dropped. Objects are only loaded in C_FinitObjectInit

Reply:
> Hi William. that call in src/lib/db.c is already dropped. Objects are only loaded in C_FinitObjectInit

Oh I missed that.